WORKSPACE

```diff
@@ -112,10 +112,10 @@ http_archive(
 # KleidiAI library, used for ARM microkernels.
 http_archive(
     name = "KleidiAI",
-    sha256 = "d8f2b5bf6eba7ab8fe3cedd97c4adc967c1befa69a6f4c4f6cbb3c102a7dd3c9",
+    sha256 = "2666071382cfe1a705aaf96d938bca4436db59fc28518e49899140b652fa9a17",
     strip_prefix = "kleidiai-32384cde728f444afdb92eecbb65e293fc6a6315",
```

Comment:
This needs to be updated as well.

Also, would it make more sense to wait for the next official release, which I think should be happening soon?

Reply:
thanks i'll update that.

I'm not supre familiar with their release plan/schedule. Do you know how soon it is? We can wait if its coming up soon (next week-ish)

i'll be on PTO next week, so i'll cc @digantdesai @GregoryComer to bring this to the finish line.

Reply:
Asked Gian Marco from Arm. Worst case can we update again? We would like to pull this kernel soon-ish.

Reply:
Ok https://gitlab.arm.com/kleidi/kleidiai/-/releases/v0.3.0 is out, let me use that.

Reply:
Created a new PR #7238 on top of this. Here is a [delta](https://github.com/mcr229/XNNPACK/compare/kleidi_p2..digantdesai:XNNPACK:kleidi_pin_bump_v0.3.0?expand=1) (ignore rebased stuff)